mmv1/products/compute/api.yaml

```diff
@@ -7468,6 +7490,60 @@ objects:
         description: |
           The IEEE 802.1Q VLAN tag for this attachment, in the range 2-4094. When
           using PARTNER type this will be managed upstream.
+      - !ruby/object:Api::Type::Array
+        name: 'ipsecInternalAddresses'
+        description: |
+          URL of addresses that have been reserved for the interconnect
+          attachment, Used only for interconnect attachment that has the
+          encryption option as IPSEC.
+
+          The addresses must be RFC 1918 IP address ranges. When creating HA
+          VPN gateway over the interconnect attachment, if the attachment is
+          configured to use an RFC 1918 IP address, then the VPN gateway's IP
+          address will be allocated from the IP address range specified
+          here.
+
+          For example, if the HA VPN gateway's interface 0 is paired to this
+          interconnect attachment, then an RFC 1918 IP address for the VPN
+          gateway interface 0 will be allocated from the IP address specified
+          for this interconnect attachment.
+
+          If this field is not specified for interconnect attachment that has
+          encryption option as IPSEC, later on when creating HA VPN gateway on
+          this interconnect attachment, the HA VPN gateway's IP address will be
+          allocated from regional external IP address pool.
+        input: true
+        min_version: beta
+        item_type: !ruby/object:Api::Type::ResourceRef
+          name: 'ipsecInternalAddress'
+          resource: 'Address'
+          imports: 'selfLink'
+          description: |
+            URL of an address that has been reserved for the interconnect
+            attachment.
+      - !ruby/object:Api::Type::Enum
+        name: 'encryption'
+        description: |
+          Indicates the user-supplied encryption option of this interconnect
+          attachment:
+
+          NONE is the default value, which means that the attachment carries
+          unencrypted traffic. VMs can send traffic to, or receive traffic
+          from, this type of attachment.
+
+          IPSEC indicates that the attachment carries only traffic encrypted by
+          an IPsec device such as an HA VPN gateway. VMs cannot directly send
+          traffic to, or receive traffic from, such an attachment. To use
+          IPsec-encrypted Cloud Interconnect create the attachment using this
+          option.
+
+          Not currently available publicly.
```

Comment:
Do we still want to include this? The schema is public but the feature is allowlisted, so I think it's a bit of a misrepresentation.